src/lib.rs

```diff
@@ -22,7 +22,7 @@ use std::{
 use tokio::runtime::Runtime;
 use up_rust::{ComparableListener, UAttributes, UCode, UListener, UPriority, UStatus, UUri};
 // Re-export Zenoh config
-pub use zenoh::config::Config;
+pub use zenoh::config as ZenohConfig;
```

Comment:
Since you're aliasing a module here, would this be better as `zenoh_config`?

Reply:
Haha, I struggled to choose between `ZenohConfig` and `zenoh_config` while using the alias name. I chose `zenoh_config` first and then renamed it to `ZenohConfig`. I'm not good at naming, so I'll follow your preference :laughing: